scala/lambdas/ingest-flow-control/src/test/scala/uk/gov/nationalarchives/ingestflowcontrol/Helpers.scala

```diff
@@ -89,15 +89,19 @@ object Helpers {
     override def writeItems[T](tableName: String, items: List[T])(using format: DynamoFormat[T]): IO[List[BatchWriteItemResponse]] = notImplemented
 
     override def queryItems[U](tableName: String, requestCondition: RequestCondition, potentialGsiName: Option[String])(using returnTypeFormat: DynamoFormat[U]): IO[List[U]] =
-      notImplemented
-
-    override def getItems[T, K](primaryKeys: List[K], tableName: String)(using returnFormat: DynamoFormat[T], keyFormat: DynamoFormat[K]): IO[List[T]] =
-      val firstPK = primaryKeys.head.asInstanceOf[IngestQueuePartitionKey]
-      errors.raise(_.getItem, "Error getting item from dynamo table") >>
+      errors.raise(_.queryItem, "Error getting item from dynamo table") >>
```

Comment:
1. Maybe "Error querying items in Dynamo table" just so that we no that we queried rather than "read"?
2. Should we add the name of the table to this message?

Reply:
changed the message. 
No need to add table name. This is a test helper, there is no actual table involved